modules/ROOT/pages/feature-flagging.adoc

```diff
@@ -747,6 +751,20 @@ This feature isn't configurable by system property.
 *Issue ID*
 
 * W-15750334
+<.^|`NTLM_AVOID_SEND_PAYLOAD_ON_TYPE_1`
```

Comment:
I think in other cases we are using the string of the property itself. In this case it would be:
`mule.ntlm.avoid.send.payload.on.type1`.